File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Grunn.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.AllArgsConstructor;
import lombok.Data;

@JsonInclude(JsonInclude.Include.NON_NULL)
@AllArgsConstructor
@Data
public class Grunn {

    private String type;

    //Annet settes kun dersom type = 99
    private String annet;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Grunn.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Grunn(
    var type: String? = null,
    //Annet settes kun dersom type = 99
    var annet: String? = null
)
